prime-router/docs/openapi.yml

```diff
@@ -127,12 +130,42 @@ components:
           type: string
           description: Does the error apply to one item
           enum:
-            - Parameter
-            - Report
-            - Item
+            - PARAMETER
+            - REPORT
+            - ITEM
         id:
           type: string
-          description: Depends on scope, either the item's id (message_id) or the report's id
+          description: Depends on scope, either the item's id (message_id) or empty
         detail:
           type: string
           description: The details of the error
+      example:
+        scope: ITEM
+        id: row0
+        details: Empty value for 'message_id'
+
+    Destination:
+      type: object
+      properties:
+        organization:
+          type: string
+          description: Entity that is receiving one or more items from this report, now or in the future.
+        organization_id:
+          type: string
+          description: Internal name that Prime uses to track the organization.
+        service:
+          type: string
+          description: An endpoint where the organization receives the data, eg, an sftp site or redox endpoint.
+        sending_at:
+          type: string
+          description: The timestamp at which this data is due to be transmitted to that organization/service.
+        itemCount:
+          type: string
+          description: The number of items (rows, in a csv) due to be sent to this organization/service.
+      example:
+        organization: Brobdingnag Public Health Department
+        organization_id: brob-phd
+        service: elr
+        sending_at: 2021-02-11T14:55-05:00
+        itemCount: 17
+
```

Comment:
I had forgotten to update the swagger doc when I made the change to `destinations` (in a different PR)